game-core/src/main/java/games/strategy/triplea/oddsCalculator/ta/OddsCalculatorPanel.java

```diff
@@ -171,18 +168,8 @@
       updateDefender(null);
       updateAttacker(null);
     }
-    if (OddsCalculatorPanel.percentageOfFreeMemoryAvailable() < 0.4) {
-      System.gc();
-      System.runFinalization();
-      System.gc();
-    }
     calculator = new ConcurrentOddsCalculator("BtlCalc Panel");
 
-    calculator.addOddsCalculatorListener(() -> {
-      calculateButton.setText("Calculate Odds");
-      calculateButton.setEnabled(true);
-    });
-
     calculator.setGameData(data);
```

Comment:
This delayed enabling of the button is actually not working, because setGameData is a blocking method AFAIK and therefore this JFrame is getting displayed after the gamedata has been loaded

Reply:
I've a large save game file at home, I can help verify the difference tonight or tomorrow. It's not too hard to find in lobby some late game's, generally anything round 15+ will be a slow data copy

Reply:
@DanVanAtta Not necessary.
I realised this asynchronous behaviour would've been broken for the default OddsCalculator, but does work without problems for the Concurrentoddscalculator. 
After realizing this I added the original behaviour back